package.json

```diff
@@ -30,7 +39,8 @@
     "jest": "^23.6.0"
   },
   "scripts": {
-    "build": "babel src --out-dir lib",
+    "compile": "comunica-compile-config config/config-default.json > lib/comunica-engine.js",
+    "build": "babel src --out-dir lib && npm run compile",
```

Comment:
Can you split into commands as follows: `build:lib` and `build:engine`? Then `build` invokes both with `npm run`.